Makefile

```
@@ -115,6 +115,13 @@ endif
 ifeq ($(UNAME_M),amd64)
 	CFLAGS += -mavx -mavx2 -mfma -mf16c
 endif
+ifeq ($(UNAME_M),ppc64)
+	POWER9_M := $(shell grep "POWER9" /proc/cpuinfo)
+	ifneq (,$(findstring POWER9,$(POWER9_M)))
+		CFLAGS += -mpower9-vector -DGGML_BIG_ENDIAN
+		CXXFLAGS += -std=c++23 -DGGML_BIG_ENDIAN
+	endif
```

It seems that this port will be big-endian even if run on non-Power9 systems (consider P8, P10, or any other generation), so I think this is more correct:

```suggestion
	ifneq (,$(findstring POWER9,$(POWER9_M)))
		CFLAGS += -mpower9-vector
	endif
	CFLAGS += -DGGML_BIG_ENDIAN
	CXXFLAGS += -std=c++23 -DGGML_BIG_ENDIAN
```